src/qdisp/CzarStats.cc

```diff
@@ -58,15 +57,18 @@ void CzarStats::setup(qdisp::QdispPool::Ptr const& qdispPool) {
 }
 
 CzarStats::CzarStats(qdisp::QdispPool::Ptr const& qdispPool) : _qdispPool(qdispPool) {
-    _histTrmitRecvRate = util::HistogramRolling::Ptr(new util::HistogramRolling(
-            "TransmitRecvRateBytesPerSec", {1'000, 1'000'000, 500'000'000, 1'000'000'000}, 1h, 10000));
-    auto bucketVals = {0.1, 1.0, 10.0, 100.0, 1000.0};
-    _histRespSetup =
-            util::HistogramRolling::Ptr(new util::HistogramRolling("RespSetupTime", bucketVals, 1h, 10000));
-    _histRespWait =
-            util::HistogramRolling::Ptr(new util::HistogramRolling("RespWaitTime", bucketVals, 1h, 10000));
+    auto bucketValsRates = {1'000.0, 1'000'000.0, 500'000'000.0, 1'000'000'000.0};
```

Comment:
Since you're not going to modify this object then do:
```
auto const ...
```

Reply:
The suggestion has been withdrawn as it won't work for the list-initializer which by itself is already `const`.